kubernetes/helm_charts/datapipeline_jobs/values.j2

```diff
@@ -680,6 +680,8 @@ assessment-aggregator:
       output.certissue.topic = {{ env_name }}.issue.certificate.request
     }
     task {
+      consumer.parallelism {{ assessaggregator_consumer_parallelism }}
```

Comment:
This needs to be consumer.parallelism = {{ assessaggregator_consumer_parallelism }}